app/controllers/idv/hybrid_mobile/capture_complete_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Idv
+  module HybridMobile
+    class CaptureCompleteController < ApplicationController
+      include IdvSession
```

Comment:
Let's see how these before actions work here. This flow is kinda of weird since it is in a different session than the one where the IdV session lives.

Reply:
Oh, maybe that's what was going wrong in the feature test. Should I go ahead and merge and then we can iterate on this when the other controller gets added?